infra/build/functions/trial_build.py

```diff
@@ -228,6 +228,7 @@ def _do_build_type_builds(args, config, credentials, build_type, projects):
       continue
 
     try:
+      logging.info('Getting info for: "%s".', project_name)
```

Comment:
Let's get rid of this, I think I added a lot of this logging when I was developing this feature, but now this output isn't so suitable for consumers.